tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -304,7 +306,7 @@ function install_artifacts() {
 
 	# Allow Mariner to use custom configuration.
 	if [ "${HOST_OS:-}" == "cbl-mariner" ]; then
-		config_path="${dest_dir}/share/defaults/kata-containers/configuration-clh.toml"
+		config_path="/host/${dest_dir}/share/defaults/kata-containers/configuration-clh.toml"
```

Comment:
Could we use `${host_install_dir}` here instead?

Reply:
We could! And I will in the follow-up.